Packages/Netherlands3D/WebServiceReader/Runtime/Scripts/WFSReader/GeoJSONGeometry.cs

```diff
@@ -0,0 +1,151 @@
+using System.Collections;
+using System.Collections.Generic;
+using Netherlands3D.Core;
+using Netherlands3D.Utilities;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Netherlands3D.WFSHandlers
+{
+    public class PointGeometry
+    {
+        public GeoJSON.GeoJSONGeometryType GeometryType;
+        public Vector3 point;
+
+        public PointGeometry(GeoJSON.GeoJSONGeometryType geometryType, Vector3 point)
+        {
+            GeometryType = geometryType;
+            this.point = point;
+        }
+    }
+
+    public class ListPointGeometry
+    {
+        public GeoJSON.GeoJSONGeometryType GeometryType;
+        public List<Vector3> points;
+
+        public ListPointGeometry(GeoJSON.GeoJSONGeometryType geometryType, List<Vector3> points)
+        {
+            GeometryType = geometryType;
+            this.points = points;
+        }
+    }
+
+    public class MultiListPointGeometry
+    {
+        public GeoJSON.GeoJSONGeometryType GeometryType;
+        public List<List<Vector3>> points;
+
+        public MultiListPointGeometry(GeoJSON.GeoJSONGeometryType geometryType, List<List<Vector3>> points)
+        {
+            GeometryType = geometryType;
+            this.points = points;
+        }
+    }
+
+    public class GeoJSONGeometry
+    {
+        public GeoJSON ActiveGeoJSON { get; set; }
+
+        public List<PointGeometry> points = new();
+        public List<ListPointGeometry> listPoints = new();
+        public List<MultiListPointGeometry> multiListPoints = new();
+
+        //private UnityEvent<Vector3> pointEvent = new UnityEvent<Vector3>();
```

Comment:
Is deze commented code -en degene hieronder- nog een work in progress? Zo niet, lekker weggooien, toch?Het staat nu in git, dus als we het ooit weer nodig hebben is een `git log` zo gedaan :)